### ISSUE TITLE
Metainfo refactor

### ISSUE THREAD

**Issue by ladinesa** [ondracka/nomad_parser_lobster#3]
Hi, 

We recently updated the metainfo definitions. I not quite sure how you want to proceed if you would like to update it yourself. I did already implement the changes if you are interested. :)

**Comment by ondracka** [comment]
Sure, if you have the changes please send a pull request.

My long-term preference would be to move the code (and also the openmx parser) to the upstream nomad repo, I stated this already in https://matsci.org/t/openmx-parser/37525 , but you suggested we wait for the metainfo refactor. So maybe maybe now would be a good time to make the move?

**Comment by ladinesa** [comment]
HI,
Great we can move both to nomad with the new metainfo definitions. However, I am not quite sure why I get a 403 when I push it to a new branch.

**Comment by ondracka** [comment]
No idea, do you need some access to this repo or is the problem at nomad side?

**Comment by ladinesa** [comment]
I just cloned your repo then tried to push my branch. My git knowledge is pretty limited so I am not sure. It says

remote: Permission to ondracka/nomad_parser_lobster.git denied to ladinesa.
fatal: unable to access 'https://github.com/ondracka/nomad_parser_lobster.git/': The requested URL returned error: 403

**Comment by ondracka** [comment]
Right, you can't push here directly now. I'm not sure what are you trying to do, if you are trying to get the metainfo changes into here, the usual way that I do it is to fork this repo from your account on github, make the changes there, push it to your repo and than make the pull request through the github interface.

I can also give you an access here, no problem.

**Comment by ladinesa** [comment]
Ah I thought that I do not anymore need permission if it is a public repo. and I also thought that pull requests are done on same repo. anyway thanks Ill push it to metainfo-refactor branch
